modules/tinymce/src/core/main/ts/content/EditorContent.ts

```diff
@@ -5,14 +5,54 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { Cell } from '@ephox/katamari';
+
+import { addDefaultGetFormats } from 'tinymce/core/content/GetContentImpl';
+import { addDefaultSetFormats } from 'tinymce/core/content/SetContentImpl';
+
+import Editor from '../api/Editor';
+import * as AddFormat from './AddFormat';
 import { Content, GetContentArgs, SetContentArgs } from './ContentTypes';
 import { getContent } from './GetContent';
 import { setContent } from './SetContent';
 
+interface EditorContent {
+  addFormat: (format: string, formatGetter: GetContentFormatter, formatSetter: SetContentFormatter) => void;
+  get: (args: Partial<GetContentArgs>) => Content;
+  set: (content: Content, args?: Partial<SetContentArgs>) => Content;
+}
+
+export type GetContentFormatter = (editor: Editor, args: GetContentArgs) => Content;
+export type SetContentFormatter = (editor: Editor, content: Content, args: SetContentArgs) => Content;
```

Comment:
`Formatter` is misleading, as it sounds like it's related to the formatter API. Can we please rename this to something else? even just changing it to `GetContentCallback` and `SetContentCallback` might be a good start?